src/components/implementation/no_interface/llbooter/llbooter.c

```diff
@@ -292,8 +280,13 @@ boot_create_cap_system(void)
 		if (boot_spd_inv_cap_alloc(h, spdid)) BUG();
 		if (boot_comp_map(h, spdid, ci, pt)) BUG();
 
-		boot_newcomp_create(spdid, new_comp_cap_info[spdid].compinfo);
-		printc("\nComp %d (%s) created @ %x!\n\n", h->id, h->name, sect->vaddr);
+		/* check for hardcoded "sl_" prefix in c obj to determine which cap image we create */
+		is_sched  = boot_check_scheduler(h->name);
```

Comment:
Really feels like these should just be flags (bitmap) associated with a scheduler.
```
typedef enum {
    BOOT_COMP_FLAG_SCHED = 1,
    BOOT_COMP_FLAG_MM = 1<<1,
} boot_comp_flag_t;
```